jobs/payment-jobs/tasks/cfs_create_invoice_task.py

```diff
@@ -291,17 +291,15 @@ def _create_pad_invoices(cls):  # pylint: disable=too-many-locals
             mailer.publish_mailer_events('pad.invoiceCreated', payment_account, additional_params)
             # Iterate invoice and create invoice reference records
             for invoice in account_invoices:
-                # Create invoice reference, payment record and a payment transaction
-                InvoiceReference.create(
+                invoice_reference = InvoiceReferenceModel(
```

Comment:
Creating invoice references this way takes 1 second at a time. I ran into a backup where we had 20,000 invoices to create.  20,000 invoice references / 60 seconds * 60 minutes = 5.5 hours to create all these invoice references. 

This reduced the time to approx 3-4 minutes.